README.md

```diff
@@ -17,8 +29,17 @@
 2. Extract the contents of this script to the same folder
 3. Run the script: `php cheat.php`
 
-## Python
+### Python
 
 0. (optional) Setup virtual env: `virtualenv env && source env/bin/activate`
 1. `pip install requests tqdm`
 2. Run the script: `python cheat.py`
+
+### Node.js
+
+1. Install the _Latest_ [Node.js](https://nodejs.org/en/) version
+2. Open the `node` folder and then open command line
```

Comment:
None of the other languages go this verbose in steps, why go the distance here?

Reply:
I was discussing with some people on discord and some of them couldn't work it out how to use it without help.

I'm happy to go overboard on other explanations as well if needbe - just want to make it accessible.

Reply:
You could make a separate PR to expand our readme though :)

Reply:
Fair point!